code/modules/mob/living/carbon/human/human_login.dm

```diff
@@ -7,4 +7,5 @@
 		to_chat(src, "<span class='notice'>You can ventcrawl! Use alt+click on vents to quickly travel about the station.</span>")
 	update_pipe_vision()
 	regenerate_icons()
+	SEND_SIGNAL(src, COMSIG_MOB_LOGIN)
```

Comment:
Can you please put this on mob/login() so its consistent with the name? Just put it at the end of it so its 100% sure the mind is initialized.

Reply:
We didn't put it here because it's only currently being registered on the human level
I actually went and double checked the profiler and login (surprisingly) was only called about 300 times per round, we can change this to be SIG_MOB_LIVING_LOGIN (or human login) if need be, thought it would be higher but it's not (Needs to be on living level or higher because minds are synced then) 
If tourte needs help with doing this, I can assist

Reply:
HUMAN_LOGIN would work for me, I just want the signal name to be accurate

Reply:
HUMAN_LOGIN added.